src/applications/vaos/actions/newAppointment.js

```diff
@@ -295,27 +293,30 @@ export function openFacilityPage(page, uiSchema, schema) {
       }
 
       if (parentId) {
-        rootOrgId = getIdOfRootOrganization(parentFacilities, parentId);
+        siteId = getSiteIdFromOrganization(
+          parentFacilities.find(parent => parent.id === parentId),
+        );
       }
 
-      facilities =
+      locations =
         newAppointment.facilities[`${typeOfCareId}_${parentId}`] || null;
 
-      if (canShowFacilities && !facilities) {
-        facilities = await getSupportedLocationsByTypeOfCare({
-          rootOrgId,
+      if (canShowFacilities && !locations) {
+        ({ locations } = await getSupportedHealthcareServicesAndLocations({
```

Comment:
What's the purpose of the parentheses surrounding this statement?

Reply:
They're required if you're destructuring into a variable that's already been defined. It's weird and annoying.